server/views/pages/csra/csraReviewPage.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "../index.njk" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% set breadCrumbs = [
+    {
+        text: 'Digital Prison Services',
+        href:  "/" | prependBaseUrl
+    },
+    {
+        text: prisonerName,
+        href: "/prisoner/" + prisonerNumber
+    },
+    {
+        text: 'CSRA history',
+        href: "#"
+    },
+    {
+        text: 'CSRA details'
```

Comment:
We don't normally show a breadcrumb for the actual page. Looks like it's not on the Figma.